README.md

```diff
@@ -307,14 +307,17 @@ release, as well as:
 
 1. **Go over the milestone and evaluate the status of any PR merging that's happening. Follow up with people on their
 bugs and fixes.** If the release introduces new bugs or regressions that can't be fixed in time, we should discuss on
-Slack about this and take a decision to go forward or postpone the release. The PR / issue owners are responsible for
-communicating any issues which might be release relevant.
+Slack about this and take a decision on how to move forward. If the issue is not ready to be merged in time, we remove issue / PR from milestone and notify the PR owner and the product manager on Slack about it. The PR / issue owners are responsible for
+communicating any issues which might be release relevant. Postponing the release should be considered as an edge case scenario based on the latest decision about [Time based releases](https://www.notion.so/rasa/Time-based-Releases-4861e429c73f402fa83dc06c0aba0435) and try to avoid it.
 
 #### Release day! 🚀
 
-1. **At the start of the day, post a small message on slack announcing release day!**. Communicate you'll be handling
+1. **At the start of the day, post a small message on slack announcing release day!** Communicate you'll be handling
 the release, and the time you're aiming to start releasing (again, no later than 4pm, as issues may arise and
-cause delays)
+cause delays). This message should be posted early in the morning and before moving forward with any of the steps of the release, 
+   in order to give enough time to people to check their PRs and Issues. That way they still have time in case they would like 
+   to add something in the Milestone before the announced release time. Template of the slack message can be found [here](https://rasa-hq.slack.com/archives/C36SS4N8M/p1613032208137500?thread_ts=1612876410.068400&cid=C36SS4N8M).
```

Comment:
The day of the release is in my opinion definitely too late to add things to the milestone. It's more that they can plan any remaining work or reach out if just tiny steps are missing for something to be merged (e.g. the CI run failed and they had to re-rerun)
```suggestion
   in order to give enough time to people to check their PRs and Issues. That way they can plan any remaining work. A template of the slack message can be found [here](https://rasa-hq.slack.com/archives/C36SS4N8M/p1613032208137500?thread_ts=1612876410.068400&cid=C36SS4N8M).
```

Reply:
in my opinion we could also directly put the Slack message here.

Reply:
I'd say we could keep the link to avoid making the instructions suuuuper long if not absolutely necessary. We have slack links elsewhere in the README anyway.